### ISSUE TITLE
Feature request

### ISSUE THREAD

**Issue by KennethFoliWork** [ffddorf/netbox-vault-secrets#14]
I am testing your plugin and i is working fine for me 
but i have a Feature request and that is that if you can add the the funktion 
so that you can add secret to Tenants for customer with secrets

**Comment by mraerino** [comment]
I feel like this is a valid request.

Can you tell me more about the use case though?
Is this for per-tenant login credentials, e.g. on DSL lines?

**Comment by KennethFoliWork** [comment]
yes it is for logins there is use for the per-tenant for logins for accounts there is global used 
we have a update system where som of the tinformation is pr tenant  and i like to store it in tenant

**Comment by danielperna84** [comment]
Hi,

I'm new to Netbox and have just stumbled upon this issue while looking for a Vault integration. That being said, I want to contribute some input to this issue:

1. Vault (Enterprise) supports the concept of [Namespaces](https://www.vaultproject.io/docs/enterprise/namespaces). If there was a boolean option to support namespaces and the namespaces in Vault match f. ex. the tenant-slugs, then this would probably be rather trivial to implement, as the tenant-slug would just have to be prefixed to the request that's already being made (I believe). However, this would not support nested Vault namespaces I assume. But still probably a step forward.
2. In my case I actually store different tenants in separate KV-storages, not utilizing the namespaces feature. I assume the `kv_mount_path` option could support a special value like `$tenant_slug` / `$tenant_id`, which replaces the typically static mount-path with `/{tenant-slug}` per request depending on the tenant of the currently viewed item.

Being new to this I don't know if those 2 options actually make sense. But I wanted to share those ideas anyways to bring this issue forward.
